.github/workflows/focus-ios-smoke

```diff
@@ -0,0 +1,153 @@
+name: "Focus iPhone Smoke Tests"
+    
+on:
+    workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    ios_version: 17.5
+    ios_simulator: iPhone 15
```

Comment:
could we add the iPad here or in the matrix somehow so that we run both in the same github action? 
We could run them in parallel I guess

Reply:
Yes, we could use matrix to run both platform in the same Github Actions.

When two or more platforms run in parallel and one fails, the whole job quit right away. I may need to add `continue-on-error` in some steps so that the test for another platform continues to run.

Reply:
yeah, you can use that, if they run in parallel one could fail and the other continue